runtime/doc/eval.txt

```diff
@@ -3771,6 +3772,10 @@ hasmapto({what} [, {mode} [, {abbr}]])			*hasmapto()*
 <		This installs the mapping to "\ABCdoit" only if there isn't
 		already a mapping to "\ABCdoit".
 
+hastabdir()						*hastabdir()*
+		The result is a Number, which is 1 when the current
+		tab has set a local path via |:tcd|, and 0 otherwise.
+
```

Comment:
No, only if a tab directory is set. For `:lcd` use `haslocaldir()`. A window can have a window-local directory and be in a tab without tab-local directory.

Personally I would have preferred changing `haslocaldir()` to return something like 0 for no local directory, 1 for window-local, 2 for tab-local and 3 for both, like a bitfield, but I think this is simpler than working with bitmasks. We can still later decide to have both.